core/dbt/contracts/graph/manifest.py

```diff
@@ -415,6 +387,7 @@ def patch_nodes(self, patches):
                     'not found or is disabled').format(patch.name)
                 )
 
+    # TODO: why is this here?
```

Comment:
👁 👁

Reply:
I really don't know. I'm going to remove it and push and if the tests pass, we'll call it good to go?